Add in story aspect section (#71)
MiniFate/MiniFate: _chapters/05-aspects-and-fate-points.md

#### Story Aspects

> Any Aspect not attached to a character is a story Aspect. Like character

Reviewer: I thought we stopped capitalizing *aspect*?

Author: We did, but I copied a lot from 3 years ago when we did... I'll fix them.

Aspects, story Aspects are always true; unlike those Aspects, story Aspects

> are dynamic. Creating an effect in the fiction is synonymous with creating a

Reviewer: dynamic... so they change? Is changing an aspect different from erasing it and writing a new one? Seems like a big point but then it doesn't go anywhere

story Aspect. Reversing that effect destroys the Aspect.

> Story Aspects can be introduced by the GM to help set a scene; for example, it
> may bear noting that ~~It's Raining Hard~~ or that ~~Everyone Seems On-Edge~~.

Reviewer: ```suggestion
Story Aspects can be introduced by the GM to help set a scene. For example,
maybe ~~It's Raining Hard~~ or ~~Everyone Seems On-Edge~~.
```

They can also be created as a result of skill rolls during gameplay. Quenching
a campfire can create ~~Pitch Blackness~~; wrestling someone to the ground
creates ~~Grappled~~.

> The destruction of Aspects is similar. They can be destroyed manually by

Reviewer: "Destroy" feels like too big a word for erasing an aspect. Is this what Evil Hat uses?

rolling; if a roll is made to break a grapple, the ~~Grappled~~ Aspect is
destroyed. Story Aspects also are destroyed when they are no longer relevant,
or no longer make sense. When the sun rises, ~~Pitch Blackness~~ is destroyed.

> Tor Byorn is leading a raiding party to steal a rival crew's cargo right out
> of their storehouse. The GM writes down the aspect ~~Abnormally Dark and
> Rainy~~ as he describes the scene. A few moments later Tor Byron heres a
> whistling shriek as a special arrow misses his head by an inch but finds its
> target in the man behind him.
> 

> > Tor Byorn's player asks the GM if he can roll `Insight` to locate the

Reviewer: ```suggestion
> Tor Byorn's player asks the GM if he can roll `Insight` to spot the
```

> > assassin, but GM declines; the aspect on the scene prevents Tor Byorn.

Reviewer: ```suggestion
> assassin, but GM declines; it wouldn't make sense with the story aspect.
```
